### ISSUE TITLE
🔥 [🐛] Not possible to customize Mute icon on ChannelList

### ISSUE THREAD

**Issue by carolinaknoll** [GetStream/stream-chat-react-native#2126]
## Issue

I am trying to customize the Mute icon that appears when a channel is muted, in order to replace the icon itself or to add a new icon beside it. However, it seems that while `ChannelList` does provide handy props for customizing the `Avatar`, title and unread sign (`PreviewAvatar`, `PreviewStatus`, `PreviewTitle`), it doesn't allow to customize the Mute icon itself. 

I [searched on the docs](https://getstream.io/chat/docs/sdk/reactnative/core-components/channel-list/#previewmutedstatus) and it seemed that it would be able to work, but there are no props on the `ChannelList` component related to mute. Is there a handy way to do that that I wasn't able to see? 😅 

Without this prop, the way I can see is to create a custom `PreviewStatus` component and have a `View` with absolute position to position the custom icon besides the existing mute icon. However, this is not ideal.

Thanks a lot for the help!

---

### Steps to reproduce

1. Have a screen with the `ChannelList` component
2. Try to add any components related to the Mute icon itself as props (like `<ChannelList PreviewMutedStatus={props => <CustomMuteComponent {...props} />} />`).
3. Notice that they are not available 🤔 

---

### Expected behavior

Ideally, we should be able to customize the ChannelList icons, such as the mute icon, and add any other icons that might match the use cases of our apps.

---

### Customization

<details><summary>Click To Expand</summary>
<p>

`PreviewAvatar`, `PreviewStatus` and `PreviewTitle` are custom components.

- ChannelList

```typescript jsx
 <ChannelList
        key={refreshKey}
        filters={{
          members: { $in: [client.user?.id] } as never,
          $or: [
            { last_message_at: { $gte: '1970-10-10T16:01:49.27673Z' } },
          ],
        }}
        sort={{ last_message_at: -1 }}
        PreviewAvatar={props => <PreviewAvatar {...props} />}
        PreviewStatus={props => <PreviewStatus {...props} />}
        PreviewTitle={props => <PreviewTitle {...props} />}
        additionalFlatListProps={{
          ListFooterComponent: FooterLoadMoreButton,
        }}
        onSelect={channel => onChannelPress(channel as never)}
      />
```
</p>
</details>

---

### Offline support

- [x] I have enabled offline support.
- [ ] The feature I'm having does not occur when offline support is disabled. (stripe out if not applicable)

---

## Environment

<details><summary>Click To Expand</summary>
<p>

#### `package.json`:

```json
# N/A
```

**`react-native info` output:**

```
System:
    OS: macOS 13.3.1
    CPU: (8) arm64 Apple M1
    Memory: 77.22 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 16.13.2 - /usr/local/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 8.1.2 - /usr/local/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.1 AI-211.7628.21.2111.8092744
    Xcode: 13.3/13E5086k - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.11 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0 
    react-native: 0.70.8 => 0.70.8 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

- **Platform that you're experiencing the issue on**:
    - [ ] iOS
    - [ ] Android
    - [ ] **iOS** but have not tested behavior on Android
    - [ ] **Android** but have not tested behavior on iOS
    - [x] Both
- **`stream-chat-react-native` 5.13.0**
- Device/Emulator info:
  - [ ] I am using a physical device
  - OS version: `Android 10`, `iOS 16.4`
  - Device/Emulator: `Android emulator`, `iPhone 14 Pro`

</p>
</details>

---

### Additional context

NA

---

### Screenshots

NA

**Comment by khushal87** [comment]
Hey @carolinaknoll, thanks for reporting this issue to us. This was indeed a bug on our side, and we have fixed this in PR #2128. Please wait until its merged and released in the next version.

**Comment by carolinaknoll** [comment]
I'll keep an eye on it. Thank you so much for fixing it so fast! 🚀
